storage/sql/peers/get.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+SELECT
+	"id",
+	"ip",
+	"wsPort",
+	"state",
+	"os",
+	"version",
+	"clock",
+	ENCODE("broadhash", 'hex') as "broadhash",
+	"height"
+FROM
+	peers
+
+${parsedFilters:raw}
```

Comment:
As soon the PR https://github.com/LiskHQ/lisk/pull/2680 is merged, need to introduce sorting here.